components/camel-stream/src/main/java/org/apache/camel/component/stream/StreamEndpoint.java

```diff
@@ -145,7 +145,8 @@ public String getHttpHeaders() {
 
     /**
      * When using stream:http format, this option specifies optional http headers, such as Accept: application/json.
-     * Multiple headers can be separated by comma.
+     * Multiple headers can be separated by comma. The format of headers should be {@code HEADER=VALUE}. An example
```

Comment:
We usually do not use @code or other formatting as the text is being extracted for tooling as well and they use plain text. So it would be good to get that removed, thanks.